build/frameworkbarrier/docker-build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# MIT License
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+BASH_DIR=$(cd $(dirname ${BASH_SOURCE}) && pwd)
+PROJECT_DIR=${BASH_DIR}/../..
+IMAGE_NAME=frameworkbarrier
+
+cd ${PROJECT_DIR}
+
+docker build -t ${IMAGE_NAME} -f ${BASH_DIR}/Dockerfile .
```

Comment:
get the return code of docker build, and return it. docker build may not success as expected, script should return non-zero code to indicate that.

Reply:
But it already has "set -o errexit"

---
In reply to: [234908308](https://github.com/Microsoft/frameworkcontroller/pull/2#discussion_r234908308) [](ancestors = 234908308)